src/vivarium_public_health/risks/exposure.py

```diff
@@ -0,0 +1,272 @@
+from abc import ABC, abstractmethod
+from typing import Any
+
+import pandas as pd
+from vivarium import Component
+from vivarium.framework.engine import Builder
+from vivarium.framework.event import Event
+from vivarium.framework.population import SimulantData
+from vivarium.framework.randomness import RandomnessStream
+from vivarium.framework.resource import Resource
+from vivarium.framework.values import Pipeline
+
+from vivarium_public_health.risks.data_transformations import get_exposure_post_processor
+from vivarium_public_health.risks.distributions import (
+    ContinuousDistribution,
+    DichotomousDistribution,
+    EnsembleDistribution,
+    PolytomousDistribution,
+    RiskExposureDistribution,
+)
+from vivarium_public_health.utilities import EntityString, get_lookup_columns
+
+_ALLOWABLE_MEASURE_TYPES = ["exposure", "coverage"]
+
+
+class Exposure(Component, ABC):
```

Comment:
Is Exposure a name that makes sense that would capture both risk and intervention? Or should it be something else?